src/huggingface_hub/utils/_subprocess.py

```diff
@@ -24,18 +24,19 @@
 
 
 def run_subprocess(
-    command: List[str], folder: str, check=True, **kwargs
+    command: Union[str, List[str]], folder: Optional[str] = None, check=True, **kwargs
 ) -> subprocess.CompletedProcess:
     """
     Method to run subprocesses. Calling this will capture the `stderr` and `stdout`,
     please call `subprocess.run` manually in case you would like for them not to
     be captured.
 
     Args:
-        command (`List[str]`):
-            The command to execute as a list of strings.
-        folder (`str`):
-            The folder in which to run the command.
+        command (`str` or `List[str]`):
+            The command to execute as a string or list of strings.
+        folder (`str`, *optional*):
+            The folder in which to run the command. Defaults to current working
+            directory (from `os.getcwd()`).
```

Comment:
@LysandreJik I slightly updated the `run_subprocess` util to make it less verbose when making a simple call. Please let me know if that could cause a problem in your opinion ? (asking since I removed a ValueError that was explicitly introduced so maybe for a good reason :) )

Reply:
No, that works well for me!

Reply:
great, thanks for confirmation :)